docs_src/examples/Ch-ExamplesAddingMQTTDevice.md

```diff
@@ -87,21 +125,27 @@ Create the device configuration file, named `my.custom.device.config.toml`, as s
   Labels = [ "MQTT", "test" ]
   [DeviceList.Protocols]
     [DeviceList.Protocols.mqtt]
+       # Comment out/remove below to use multi-level topics
        CommandTopic = "CommandTopic"
+       # Uncomment below to use multi-level topics
+       # CommandTopic = "command/my-custom-device"
     [[DeviceList.AutoEvents]]
        Interval = "30s"
        OnChange = false
        SourceName = "message"
 ```
 
-- `CommandTopic` is used to publish the GET or SET command request
+!!! note
+    `CommandTopic` is used to publish the GET or SET command request 
+    
+    **Using Multi-level Topic:** Remember to set `CommandTopic = "command/my-custom-device"`.
 
 ## Prepare docker-compose file
 
 1. Clone edgex-compose
 ```
 $ git clone git@github.com:edgexfoundry/edgex-compose.git
-$ git checkout ireland
+$ git checkout jakarta
```

Comment:
Humm, I think we need a note to use `main` until `Jakarta` is released.

Reply:
done